test/test_params.py

```diff
@@ -0,0 +1,34 @@
+test_params = {
```

Comment:
Let's move it into `test_api_responses.py` as it is only used there and the test's code is pretty small anyways.